### ISSUE TITLE
Add a link to github?

### ISSUE THREAD

**Issue by karussell** [graphhopper/directions-api-doc#19]
Should we include a link inside the documentation to this repo?

**Comment by oblonski** [comment]
Yes, we can do so explicitly. However, if you click on our logo, you will be directed to here.

**Comment by karussell** [comment]
Ah, ok. But usually on these documentation sites I look for a fork or github icon or just "edit on github" link. Also shouldn't the graphhopper logo link to graphhopper.com like we do on discuss and our dashboard?

**Comment by oblonski** [comment]
Ok. Linked GraphHopper now. Makes sense. Now we need a place to link the github repo of our docs.

**Comment by karussell** [comment]
:+1: 

We could use something like:

<a href="https://github.com/graphhopper/directions-api-doc/edit/master/spec/openapi.yaml"><img style="position: absolute; top: 0; right: 0; border: 0; width: 149px; height: 149px;" src="http://aral.github.com/fork-me-on-github-retina-ribbons/right-graphite@2x.png" alt="Fork me on GitHub"></a>

From [here](https://2018.ar.al/scribbles/fork-me-on-github-retina-ribbons/)
